tutor/src/screens/assignment-grade/question.js

```diff
@@ -94,7 +94,11 @@ const ExpandGraded = observer(({ ux }) => {
         {ux.expandGradedAnswers ? 'Hide' : 'Expand'} graded answers {gradeProgress}
       </Button>
       <label>
-        Average Score: {ScoresHelper.formatPoints(ux.selectedHeading.averageGradedPoints)} out of {ScoresHelper.formatPoints(ux.selectedHeading.responseStats.availablePoints)}
+        Average Score: {
+          ux.selectedHeading.dropped
+            ? ScoresHelper.formatPoints(ux.getDropQuestionPoints)
+            : ScoresHelper.formatPoints(ux.selectedHeading.averageGradedPoints)
+        } out of {ScoresHelper.formatPoints(ux.selectedHeading.points_without_dropping)}
       </label>
```

Comment:
not a big deal but since you're calling `formatPoints` on both values you could move the ternary inside the call like this:
```
ScoresHelper.formatPoints(
  ux.selectedHeading.dropped ? ux.getDropQuestionPoints : ux.selectedHeading.averageGradedPoints
)
```